docs/ref/databases.txt

```diff
@@ -367,28 +367,6 @@ this for you (the database backend layer and the model population layer are
 separated internally so the database layer doesn't know it needs to make this
 conversion in this one particular case).
 
-If you're using MySQLdb 1.2.1p2, Django's standard
```

Comment:
There's a reference in the previous paragraph to "MySQLdb 1.2.2" which might mean "1.2.2 and newer" (original commit: f2b389b354165cceb578aa3b13bec88f0e44c654).